apps/prairielearn/src/migrations/20240501185744_questions__shared_publicly_source__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE questions
+ADD COLUMN shared_publicly_source BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Bikeshedding: `shared_publicly_with_source`? `source_shared_publicly`? Something else? The current column name feels strange to me. Let's discuss.

Reply:
I like `shared_publicly_with_source` to keep the column sorted near the existing `shared_publicly`.

Reply:
I agree `shared_publicly_with_source` is best, thanks for the suggestion @nwalters512 . The alphabetic sorting thing is exactly the reason I didn't do `source_shared_publicly` in the first place.